aea/cli_gui/__init__.py

```diff
@@ -335,12 +335,14 @@ def start_agent(agent_id: str, connection_id: PublicId):
     app_context.agent_tty[agent_id] = []
     app_context.agent_error[agent_id] = []
 
+    # we don't seem to ever join this
```

Comment:
@solarw any suggestion how to best manage this? Maybe by attaching to app context?